core/src/main/kotlin/org/evomaster/core/output/oracles/ImplementedOracle.kt

```diff
@@ -85,8 +87,16 @@ abstract class ImplementedOracle {
      */
 
     fun retrievePath(objectGenerator: ObjectGenerator, call: RestCallAction): PathItem? {
+        val serverUrl = objectGenerator.getSwagger().servers[0].url
+        val basePath: String = try {
+            URI(serverUrl).path.trim()
+        } catch (e: URISyntaxException){
+            LoggingUtil.uniqueWarn(log, "Invalid URI used in schema to define servers: $serverUrl")
+            ""
+        }
```

Comment:
this is copied&pasted from `addActionsFromSwagger`.
this piece of code should be refactored out in a static function, and then use such function in both places to avoid code-duplication